pdp/static/js/pdp_raster_map.js

```diff
@@ -55,18 +97,23 @@ function dasToUnitsSince(data) {
         dateString = m[3],
         sDate;
 
+    var calendar;
+    reg = /calendar \"(standard|gregorian|365_day|noleap|360_day)\"/,
+    m = reg.exec(s),
+    calendar = m[1];
```

Comment:
I considered adding a similar default value for "units" in the same function, but wasn't entirely sure whether "days" was a good default.